lib/containers/container_images.pm

```diff
@@ -208,10 +208,10 @@ sub test_zypper_on_container {
         record_info('bsc#1192941', 'bsc#1192941 - zypper-docker entrypoint confuses program arguments');
     }
     validate_script_output("$runtime run -i --entrypoint '' $image zypper lr -s", sub { m/.*Alias.*Name.*Enabled.*GPG.*Refresh.*Service/ }, timeout => 180);
-    script_retry("$runtime run -i --name 'refreshed' --entrypoint '' $image zypper -nv ref", timeout => 300, retry => 3, delay => 60);
+    assert_script_run("$runtime run -i --name 'refreshed' --entrypoint '' $image zypper -nv ref", timeout => 600);
     assert_script_run("$runtime commit refreshed refreshed-image", timeout => 120);
     assert_script_run("$runtime rm -f refreshed");
-    script_retry("$runtime run -i --name 'refreshed-image' --rm --entrypoint '' $image zypper -nv ref", timeout => 300, retry => 3, delay => 60);
+    assert_script_run("$runtime run -i --name 'refreshed-image' --rm --entrypoint '' $image zypper -nv ref", timeout => 600);
```

Comment:
I think this should be 
```
$runtime run -i --rm  --entrypoint '' localhost/refreshed-image zypper -nv ref
```
Otherwise, we are just repeating the same command as in line 211...  I guess, what we want to do is to check if `zypper ref` is "faster" in the refreshsed-container...

Reply:
We check if the `zypper` is available also in derivative image.